### DISCUSSION TITLE
Bootstrap styles for data frame (instead of kable, knitr)

### DISCUSSION THREAD

**Discussion by sarah-h-cohen** [quarto-dev/quarto-cli#654] [Q&A] [answered]
Is it possible to assign Bootstrap classes to the tables produced from a code chunk? I saw how to do it in the context of the markdown, but wasn't sure it was possible in results. (This might be an alternative to kable / knitr tables even though they're not paginated.)

**Comment by jjallaire** [comment, marked as answer]
We currently automatically assign the `table-sm` and `table-striped` classes to markdown tables (including ones in results). In turn, kable should be producing markdown tables by default so you should see these classes applied (unless you've explicitly turned on HTML output or are using e.g. kableExtra). It's on our list to make these classes customizable (e.g. opt out of these styles and/or add other styles).
